tests/test_loky_module.py

```diff
@@ -19,9 +19,13 @@ def test_cpu_count():
     assert type(cpus) is int
     assert cpus >= 1
 
+    cpus_physical = cpu_count(maybe_physical_only=True)
+    assert type(cpus_physical) is int
+    assert 1 <= cpus_physical <= cpus
+
 
 cpu_count_cmd = ("from loky.backend.context import cpu_count;"
-                 "print(cpu_count())")
+                 "print(cpu_count({}))")
```

Comment:
It might be easier to understand the test if you give a name to the format string.
```suggestion
                 "print(cpu_count({args}))")
```